specification/desktopvirtualization/resource-manager/readme.md

```diff
@@ -52,6 +52,11 @@ input-file:
 - Microsoft.DesktopVirtualization/preview/2019-09-24-preview/desktopvirtualization.json
 ```
 
+``` yaml

```

Comment:
There should only be one default tag:. Please remove the other two:
`
``` yaml
openapi-type: arm
tag: package-2019-01-23-preview
``` 
``` yaml
openapi-type: arm
tag: package-2019-09-24-preview
```
`

Reply:
Also please remove ## <Lang> section if there is no corresponding readmd.<lang>.md files. Such as C#, and Python.

Are you missing readme.python.md?

Reply:
Also for each language md, please double check namespace and output path for correctness.

Reply:
> Also please remove ## section if there is no corresponding readmd..md files. Such as C#, and Python.
> 
> Are you missing readme.python.md?

they are all there. Both C# and python. Not sure why you don't see it.

Reply:
> Also for each language md, please double check namespace and output path for correctness.

seems like all the languages have a different output path, and namespace. How do I determine if they're correct?